incident/mysql/incident_test.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mysql
+
+import (
+	"bytes"
+	"context"
+	"database/sql"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/golang/glog"
+
+	_ "github.com/go-sql-driver/mysql" // Load MySQL driver
+)
+
+type entry struct {
+	baseURL, summary, category, fullURL, details string
+}
+
+func checkContents(ctx context.Context, t *testing.T, want []entry) {
+	t.Helper()
+
+	tx, err := testDB.BeginTx(ctx, nil /* opts */)
+	if err != nil {
+		t.Fatalf("failed to create state transaction: %v", err)
```

Comment:
Why do you use the word "state" in these error messages?

Reply:
Copy pasta, removed.